docs/modules/transform-sync/how-it-works.md

```diff
@@ -0,0 +1,37 @@
+# How the Transform Synchronization Feature Module works
+
+## What is Transform
+
+There are two concepts of the location of a SpatialOS entity:
```

Comment:
probably needs a note that this is not the same as the Transform component offered by Unity, though it does do some conversion between unity and spatialos